pkg/querier/blocks_store_queryable.go

```diff
@@ -781,28 +813,60 @@ func (q *blocksStoreQuerier) fetchSeriesFromStores(ctx context.Context, sp *stor
 				if s := resp.GetStats(); s != nil {
 					indexBytesFetched += s.FetchedIndexBytes
 				}
-			}
 
-			numSeries := len(mySeries)
-			chunksFetched, chunkBytes := countChunksAndBytes(mySeries...)
+				if ss := resp.GetStreamingSeries(); ss != nil {
```

Comment:
Should we / can we enforce the chunks limit here for streaming series, similar to what we do in https://github.com/grafana/mimir/pull/5369? Not sure if this is possible to know early in store-gateways.

Reply:
I think we can, but I would prefer doing that in a follow up PR